### PR TITLE
Convert to v2 addon

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
depends on https://github.com/CrowdStrike/ember-velcro/pull/19

Note that  2,865 of the changes are from the lockfile.

So this PR is more deletions that overall additions, if you omit the lockfile 🙈

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  install_dependencies:
+    name: Install
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/pnpm
+
+
+
+
```
Comment (danwenzel) [inline .github/workflows/ci.yml:28]: nit: any reason for the extra newlines in this file?

**Review by danwenzel — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (NullVoxPopuli) [reply .github/workflows/ci.yml:28]: the file is generated from: https://github.com/NullVoxPopuli/ember-ci-update

**Review by ynotdraw — APPROVED** [review]
🎉

File: package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
     "start:addon": "pnpm --filter ember-velcro start --no-watch.clearScreen",
     "test": "pnpm --filter test-app test",
     "lint": "pnpm --filter '*' lint",
-    "lint:fix": "pnpm --filter '*' lint:fix"
+    "lint:fix": "pnpm --filter '*' lint:fix",
```
Comment (nicolechung) [inline package.json:12]: Issue: we should update the [README](https://github.com/CrowdStrike/ember-velcro#installation) to reflect that we should `pnpm install`

Comment (nicolechung) [reply package.json:12]: Also the [CONTRIBUTING](https://github.com/CrowdStrike/ember-velcro/blob/main/CONTRIBUTING.md#installation)

I don't see a yarn lockfile so I don't think that is up to date

**Comment by nicolechung** [comment]
I don't know if this matters, but I am seeing this error:
Screen Shot 2022-10-05 at 7 03 43 PM

**Comment by nicolechung** [comment]
Not sure if it's in the scope of this PR, but when I hit "format on save" in my editor, it causes problems that eslint complains about:

Screen Shot 2022-10-05 at 7 13 04 PM

The prettierrc only has single quotes turned on though (that seems to be working for me, it switches `"` to `'` correctly on save.

File: package.json (hunk 2 above)
Comment (NullVoxPopuli) [reply package.json:12]: updated, thanks!

File: README.md [hunk 3]
```diff
@@ -92,7 +92,13 @@ Installation
 ------------------------------------------------------------------------------
 
 ```
-ember install github:camskene/ember-velcro
+npm install ember-velcro
```
Comment (nicolechung) [inline README.md:95]: Nit: (sorry you were so fast - I missed this!) OR maybe this should be a follow up 

The current test instructions in the CONTRIBUTING only work if you `cd test-app`, should these be the pnpm instructions instead?

I ran the linting instructions and they work for me though.

**Comment by NullVoxPopuli** [comment]
> I don't know if this matters, but I am seeing this error:

I don't think that ever worked, just by looking at the code. 🤔 
The component yields a hash, but the usage is expecting two separate positional modifiers

> but when I hit "format on save" in my editor, it causes problems that eslint complains about:

Maybe we need to review your editor settings, cause my editor doesn't do this? 🤔

File: README.md (hunk 3 above)
Comment (NullVoxPopuli) [reply README.md:95]: oh good callout 

I should copy from the blueprint, https://github.com/embroider-build/addon-blueprint/blob/main/files/CONTRIBUTING.md

**Review by nicolechung — APPROVED** [review]
Looks good to me. I ran all the package.json scripts (except for the ci:update one)

**Comment by nicolechung** [comment]
>Maybe we need to review your editor settings, cause my editor doesn't do this? 🤔

Oh yeah, I will take a look at this again...maybe it's ignoring using eslint to format.

**Pull request merged**